sqrl/graphql/__init__.py

```diff
@@ -1,14 +1,20 @@
 from flask import Flask
 from graphql_server.flask import GraphQLView
+from graphql.validation.rules.custom.no_schema_introspection import NoSchemaIntrospectionCustomRule
+from graphql.validation.specified_rules import specified_rules
 from sqrl.graphql.schema import schema
 
 
 def init_app(app: Flask) -> None:
     """Initialise GraphQL with a flask app context."""
+    on_dev = app.config.get('ENV') == 'development'
```

Comment:
Can you add a copy of this to the .env.example?

Reply:
this is already in .env.example